### ISSUE TITLE
Please make sure the kmod spl devel package for your distribution

### ISSUE THREAD

**Issue by votdev** [OpenMediaVault-Plugin-Developers/openmediavault-zfs#33]
I'm not able to install the package because zfs-dkms fails with the following error message:

```
------------------------------
Deleting module version: 0.6.5.11
completely from the DKMS tree.
------------------------------
Done.
Entpacken von zfs-dkms (0.6.5.11-1~bpo9+1) über (0.6.5.11-1~bpo9+1) ...
zfs-dkms (0.6.5.11-1~bpo9+1) wird eingerichtet ...
Loading new zfs-0.6.5.11 DKMS files...
Building for 4.13.0-0.bpo.1-amd64
Building initial module for 4.13.0-0.bpo.1-amd64
configure: error: 
	*** Please make sure the kmod spl devel package for your distribution
	*** is installed then try again.  If that fails you can specify the
	*** location of the spl source with the '--with-spl=PATH' option.
Error! Bad return status for module build on kernel: 4.13.0-0.bpo.1-amd64 (x86_64)
Consult /var/lib/dkms/zfs/0.6.5.11/build/make.log for more information.
openmediavault-zfs (4.0) wird eingerichtet ...
```

**Comment by ryecoaaron** [comment]
The 0.6.5.11 module doesn’t build with the 4.13 kernel. I built the zfs 0.7.3 packages from Debian Sid and put them in the omv-extras testing repo. These will compile against the 4.13 kernel.  Once these packages had enough testing, I would move the zfs packages to the regular repo unless Debian released packages that did compile against the 4.13 kernel.

**Comment by ryecoaaron** [comment]
This shouldn't be an issue anymore since 0.7.6 is in the repo.
